package.json

```diff
@@ -15,24 +18,27 @@
     "prettier": "prettier --write 'src/**/*.ts' 'tests/**/*.ts'"
   },
   "dependencies": {
+    "@aws-sdk/client-s3": "^3.631.0",
     "@evervault/sdk": "^4.3.0",
-    "axios": "^1.4.0",
-    "body-parser": "^1.20.2",
+    "axios": "^1.7.3",
```

Comment:
Is this the version of axios that was patched against the SSRF vulnerability?

Reply:
`npm audit` doesn't show high or critical vulnerabilities for this version, however I will update to latest version

Reply:
`axios` has been updated to version `1.7.7`